doc/usecases/gamification/gain_blossoms.md

```diff
@@ -0,0 +1,25 @@
+# Use Case: Gain Blossoms
+
+## Summary
+
+- **Scope:** Gain Blossoms
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user is awarded with Blossoms for achieving certain milestones.
+- **Status:** Draft
+
+## Scenarios
+
+- **Precondition:**
+  - The user has opened the app and uses it.
+- **Main success scenario:**
+  - The user can see a list of incompleted milestones and their respective requirements, like e.g. planting full groups of companions or collaborating with other users on their map, in the users profile.
```

Comment:
Should milestones always refer to the first time a user uses a feature or should we also include follow-up milestones with higher amounts of the same requirement, like e.g. after planting their first plant the next step would be planting 10 plants etc.? Should not be that much of a step up difficulty wise for us but could increase long term engagement with the milestones system.

Reply:
I like the idea. One important thing: The milestones should (similarly) repeat every year, this is also important to note in the use case. This is because some plans need to be refined or redone every season.